projects/observability/src/shared/components/cartesian/d3/chart/cartesian-chart.ts

```diff
@@ -80,6 +84,10 @@ export class DefaultCartesianChart<TData> implements CartesianChart<TData> {
     this.tooltip && this.tooltip.destroy();
     this.legend && this.legend.destroy();
 
+    if (this.activeSeriesSubscription) {
```

Comment:
why not just `this.activeSeriesSubscription?.unsubscribe();`

Reply:
Fixed